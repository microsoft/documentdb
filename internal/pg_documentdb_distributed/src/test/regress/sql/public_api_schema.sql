-- show all functions exported in helio_api_distributed
\df helio_api_distributed.*

-- show all aggregates exported
\da+ helio_api_distributed.*

-- This is last (Tables/indexes)
\d helio_api_distributed.*
