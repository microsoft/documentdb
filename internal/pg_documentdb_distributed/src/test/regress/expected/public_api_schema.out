-- show all functions exported in helio_api_distributed
\df helio_api_distributed.*
                                       List of functions
        Schema         |        Name        | Result data type |  Argument data types   | Type 
---------------------------------------------------------------------
 helio_api_distributed | complete_upgrade   | boolean          |                        | func
 helio_api_distributed | initialize_cluster | void             |                        | func
 helio_api_distributed | rebalancer_start   | helio_core.bson  | p_spec helio_core.bson | func
 helio_api_distributed | rebalancer_status  | helio_core.bson  | p_spec helio_core.bson | func
 helio_api_distributed | rebalancer_stop    | helio_core.bson  | p_spec helio_core.bson | func
(5 rows)

-- show all aggregates exported
\da+ helio_api_distributed.*
                     List of aggregate functions
 Schema | Name | Result data type | Argument data types | Description 
---------------------------------------------------------------------
(0 rows)

-- This is last (Tables/indexes)
\d helio_api_distributed.*
      Table "helio_api_distributed.helio_cluster_data"
  Column  |      Type       | Collation | Nullable | Default 
---------------------------------------------------------------------
 metadata | helio_core.bson |           | not null | 
Indexes:
    "helio_cluster_data_pkey" PRIMARY KEY, btree (metadata)
Triggers:
    helio_versions_trigger AFTER DELETE OR UPDATE ON helio_api_distributed.helio_cluster_data FOR EACH STATEMENT EXECUTE FUNCTION helio_api_internal.update_helio_version_data()

Index "helio_api_distributed.helio_cluster_data_pkey"
  Column  |      Type       | Key? | Definition 
---------------------------------------------------------------------
 metadata | helio_core.bson | yes  | metadata
primary key, btree, for table "helio_api_distributed.helio_cluster_data"

